src/python/python.cpp

```diff
@@ -275,7 +278,8 @@ struct PyGeneratorParams {
   }
 
   pybind11::array_t<int32_t> py_input_ids_;
-  pybind11::array_t<float> py_whisper_input_features_;
+  pybind11::array py_whisper_input_features_;
```

Comment:
Can we add a TODO comment here that this needs to be renamed since we will do that in a subsequent PR?